addon/src/motions/adjust-color.ts

```diff
@@ -1,20 +1,35 @@
 import { rAF } from '../-private/concurrency-helpers';
-import Motion from '../-private/motion';
+import Motion, { BaseOptions } from '../-private/motion';
+import Sprite, { CopiedCSS } from '../-private/sprite';
```

Comment:
@wagenet should we do here `import type Sprite, { CopiedCSS } from '../-private/sprite';`?